### PR TITLE
Prepare 1.1.0

### PR DESCRIPTION
**Pull request by nikku**
Prepare upcoming release.

----

This proposes prefixing links to fixed issues with actual repositories:

image

### CODE REVIEW

**Comment by marstamm** [comment]
+1 for prefixing links to fixed issues with actual repositories

I would propose we just get rid of the link text completely, as [Github will shorten links](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls#issues-and-pull-requests) to issues automatically when viewed in the browser. This approach would be less error prone to typos during creation and ensure a consistent style throughout our changelogs in the future.

Potential downside: links to other repos will always have the org in the shortened link, making them a bit longer

image

**Comment by nikku** [comment]
> I would propose we just get rid of the link text completely, as [Github will shorten links](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls#issues-and-pull-requests) to issues automatically when viewed in the browser.

This is a very new feature for Markdown files, and it also does not work for commits yet:

image

I'd rather propose to invest into some automation that unfurls these URL references and turns them into nicely visible rendered Markdown.

**Comment by nikku** [comment]
> I'd rather propose to invest into some automation that unfurls these URL references and turns them into nicely visible rendered Markdown.

One option here: https://github.com/camunda/camunda-bpmn-js/pull/228.

**Comment by marstamm** [comment]
Let's move the Merkdown discussion to https://github.com/camunda/camunda-bpmn-js/pull/228 to we are not blocked with this release

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,19 @@ All notable changes to [camunda-bpmn-js](https://github.com/camunda/camunda-bpmn
...
 ___Note:__ Yet to be released changes appear here._
 
+## 1.1.0
+
+* `DEPS`: update to `bpmn-js-properties-panel@1.15.0`
+* `DEPS`: update to `bpmn-js@11.1.0`
+
+### Key changes in Modeling
+
+
```
Comment (marstamm) [inline CHANGELOG.md:16]: Let's not have empty sub-headings

```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,19 @@ All notable changes to [camunda-bpmn-js](https://github.com/camunda/camunda-bpmn
 
 ___Note:__ Yet to be released changes appear here._
 
+## 1.1.0
+
+* `DEPS`: update to `bpmn-js-properties-panel@1.15.0`
+* `DEPS`: update to `bpmn-js@11.1.0`
```
Comment (marstamm) [inline CHANGELOG.md:12]:
```suggestion
* `DEPS`: update to `bpmn-js@11.1.1`
```

**Comment by nikku** [comment]
Thanks for the follow-up review. Incorporated all your changes.

**Pull request merged**